test/e2e_node/runner/local/run_local.go

```diff
@@ -66,6 +66,14 @@ func main() {
 		systemSpecFile := filepath.Join(rootDir, system.SystemSpecPath, *systemSpecName+".yaml")
 		args = append(args, fmt.Sprintf("--system-spec-name=%s --system-spec-file=%s --extra-envs=%s", *systemSpecName, systemSpecFile, *extraEnvs))
 	}
+	// prepare to start kubelet with swap off, avoid ginkgo node timeout
+	swapoff, err := exec.LookPath("swapoff")
+	if err != nil {
+		klog.Exitf("Test failed: %v", err)
+	}
+	if err := runCommand(swapoff, "-a"); err != nil {
```

Comment:
Do we run tests as a privileged user? `swapoff` needs to be run as root in order to not throw an error:

```
ehashman@red-dot:~$ swapoff -a
swapoff: Not superuser.
```